extensions/ccui/uiwidgets/scroll-widget/UIPageView.js

```diff
@@ -495,6 +495,8 @@ ccui.PageView = ccui.Layout.extend(/** @lends ccui.PageView# */{
      * @param {cc.Touch} touch
      */
     interceptTouchEvent: function (eventType, sender, touch) {
+        if (!this.isTouchEnabled()) return;
```

Comment:
Hi @mutoo ,

There is a PR about `interceptTouchEvent`,  https://github.com/cocos2d/cocos2d-x/pull/8949
We will merge this pr after he modified, please modify the commit `fix interceptTouchEvent;`.

Thanks 
David